### PR TITLE
Added size unit for Content size and File size filters

### PR DESCRIPTION
**Pull request by ankitgoyalgloballogic**
#REF: #MOBILEAPPS-1041

### CODE REVIEW

File: ContentApp/Modules/Advance Search/Number Range/SearchNumberRangeComponentViewModel.swift [hunk 1]
```diff
@@ -26,7 +26,8 @@ class SearchNumberRangeComponentViewModel: NSObject {
     let maxCharacters = 9
 
     var title: String {
-        return NSLocalizedString(selectedCategory?.name ?? "", comment: "")
+        let name = NSLocalizedString(selectedCategory?.name ?? "", comment: "")
+        return String(format: "%@ (KB)", name)
```
Comment (aman-alfresco) [inline ContentApp/Modules/Advance Search/Number Range/SearchNumberRangeComponentViewModel.swift:30]: please define in the string file for localization

Comment (ankitgoyalgloballogic) [reply ContentApp/Modules/Advance Search/Number Range/SearchNumberRangeComponentViewModel.swift]: Done

File: ContentApp/BusinessLayer/Lists/Models/SearchFacetFields.swift [hunk 2]
```diff
@@ -57,6 +64,15 @@ class SearchFacets: NSObject {
         }
         return count
     }
+    
+    private static func isFileSizeFacet(for label: String?) -> Bool {
+        let fileSizeLocalizedString = NSLocalizedString("SEARCH.FACET_FIELDS.SIZE", comment: "")
```
Comment (gl-lovekesh) [inline ContentApp/BusinessLayer/Lists/Models/SearchFacetFields.swift:69]: Need to compare the key not value for the key.

File: ContentApp/Modules/Advance Search/List with Search/SearchFacetListComponentViewModel.swift [hunk 3]
```diff
@@ -57,9 +57,17 @@ class SearchFacetListComponentViewModel {
             return (0.0, 0.0)
         }
     }
+    
+    func isFileSizeFacet(for label: String?) -> Bool {
```
Comment (gl-lovekesh) [inline ContentApp/Modules/Advance Search/List with Search/SearchFacetListComponentViewModel.swift:61]: Reuse it as used before as used.

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]

**Review by aman-alfresco — APPROVED** [review]

**Pull request merged**